tests/test_deploy_github_pages.py

```diff
@@ -391,8 +408,8 @@ def test_abort_local_committed_changes_exist_in_source_branch(setup_test_env):
                                                  "--source_branch", source_branch,
                                                  "--module_path", "../test_package", "../another_test_package"])
     assert e.match(f"Abort. Local commit id .* and commit id of remote source branch"
-                   f" .* are not equal. Please push your changes or pull new commits from remote.")
-    assert e.type == SystemExit
+                   f" .* are not equal. Please push your changes or pull new commits from remote.") \
+           and e.type == SystemExit
```

Comment:
You may also wan't to consider making it a `multi line assert` using the `()` syntax.